flexmeasures/utils/config_utils.py

```diff
@@ -57,9 +57,7 @@ def configure_logging():
 
 def check_app_env(env: str | None):
     if env not in (
-        "documentation",
         "development",
-        "testing",
         "staging",
         "production",
     ):
```

Comment:
The following print statement would need to be updated correspondingly, I suppose:
```
        print(
            f'Flexmeasures environment needs to be either "documentation", "development", "testing", "staging" or "production". It currently is "{env}".'
        )
```